seahorn/lib/qsort_override.c

```diff
@@ -0,0 +1,18 @@
+
+#include <stdint.h>
+#include <stdlib.h>
+#include <seahorn/seahorn.h>
+#include <nondet.h>
+
+#define MEM_IS_ACESSABLE(base, len) (((len) == 0) || (base))
+
+void qsort(void *base, size_t num, size_t size, int (*compare)(const void *, const void *)) {
```

Comment:
why can't we call the real implementation of `qsort` here? The advantage of symbolic execution is that we can interact with actual code during it.